src/fe/implicit/_.js

```diff
@@ -11,6 +16,7 @@ export const curry2 = func =>
     return func.call(null, arg1, arg2);
   };
 
+// remaining
```

Comment:
Copy documentation from curry2, and change the two arguments part to three arguments.